lib/data/monorepo.json

```diff
@@ -560,7 +560,7 @@
     "strum": "https://github.com/Peternator7/strum",
     "stryker-js": "https://github.com/stryker-mutator/stryker-js",
     "stylex-swc": "https://github.com/Dwlad90/stylex-swc-plugin",
-    "surveyjs": "https://github.com/surveyjs/surveyjs",
+    "surveyjs": "https://github.com/surveyjs/survey-library",
```

Comment:
We probably want to keep both for some time. How long ago was it renamed?

Reply:
I believe they have retained the git history so it's difficult to say when it was renamed, but it has definitely been some time.
How would you suggest to keep both? E.g have `surveyjs` and `survey-library`?

Ideally https://github.com/surveyjs/survey-creator would also be included in the group, do you have any idea how that could be possible?

Reply:
Thanks, I updated now to group all three under `surveyjs`.

`survey-library` and `survey-creator` share release versions and must be updated together, see their release tags: 
- https://github.com/surveyjs/survey-library/tags
- https://github.com/surveyjs/survey-creator/tags